front/app/components/UI/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { colors } from 'utils/styleUtils';
+
+// components
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import Link from 'utils/cl-router/Link';
+
+// types
+type SingleBreadcrumbProps = {
+  label: string;
+  linkTo?: string;
+  isCurrentPage?: boolean;
+};
+
+type BreadcrumbsProps = {
+  breadcrumbs: SingleBreadcrumbProps[];
+};
+
+const SingleBreadcrumb = ({
+  label,
+  linkTo,
+  isCurrentPage,
```

Comment:
Optional boolean props should always have a default value (imo). I've been surprised by behavior in the past when I expected a prop wasn't used, then forgot undefined means/converts to false. So I would make it explicit with isCurrentPage = false.